cli/command/system/inspect.go

```diff
@@ -214,3 +210,9 @@ func inspectAll(ctx context.Context, dockerCli *command.DockerCli, getSize bool,
 		return nil, nil, errors.Errorf("Error: No such object: %s", ref)
 	}
 }
+
+func isErrSkippable(err error) bool {
+	return apiclient.IsErrNotFound(err) ||
+		strings.Contains(err.Error(), "not supported") ||
```

Comment:
Wondering if this was actually checking for a "not implemented" error; https://github.com/moby/moby/blob/3a633a712c8bbb863fe7e57ec132dd87a9c4eff7/client/errors.go#L121

(not for this PR, but we should look at that)